apps/client/src/app/import/import.component.ts

```diff
@@ -596,7 +597,13 @@ export class ImportComponent implements OnInit {
 
   public importDisabled(): boolean {
     if (this.activeTab === 'file') {
-      return !this.files || this.files.length === 0 || !this.importBundle || !this.importBundle.entry || this.importBundle.entry.length === 0;
+      let isDisabled = !this.files || this.files.length === 0 || !this.importBundle || !this.importBundle.entry || this.importBundle.entry.length === 0;
+      if (isDisabled && this.files.length > 0) {
```

Comment:
Even though there are entries in `this.files` doesn't necessarily guarantee that the error is related to XLSX. `populateFile()` is responsible for determine how to process the drag-and-dropped/selected file. There is logic specific to XLSX in there. I suspect when you upload an invalid XLSX `this.importService.convertExcelToValueSetBundle(result)` is returning `success = false`. Maybe display the error message when that happens, instead of during the "isDisabled" checking logic. We may, in the future, add more errors messages that are specific to other types of imports. So, not all error messages are related to XLSX imports.

Reply:
i agree, much better approach...fixing now

Reply:
i've pushed changes. 
1. fixed convertExcelToValueSetBundle() to return success = false if the sheet doesn't have any rows. I can fix whatever else needs fixing for another invalid excel but this works for now for a blank excel which is obviously wrong.
2.  If the covert method throws a valuation error I don’t add it to the list of files. 
3.  If an error was thrown on the convert method and we show an error message to the user, if they upload a valid file it goes away.